.changeset/few-panthers-arrive.md

```diff
@@ -0,0 +1,7 @@
+---
+"wagmi": minor
+"@wagmi/core": minor
+"@wagmi/cli": minor
```

Comment:
TypeScript doesn't follow semver and there aren't any breaking changes coming from v4.7.4 (previous minimum version) so making this a minor.